modules/storage-persistent-storage-pv.adoc

```diff
@@ -41,11 +41,11 @@ once it is released.
 // - OpenStack Cinder
 - AWS Elastic Block Store (EBS)
 - Azure Disk
-- Azure File
```

Comment:
Why is `Azure File` removed ?

Reply:
@liangxia Seems to have been a merge conflict that I thought was fixed. `Azure File` should not be removed. Will try to resolve in this PR but may need to open a new PR. Thanks.

Reply:
@liangxia Seems to have been a merge conflict that I thought was fixed. Azure File should not be removed. Will try to resolve in this PR but may need to open a new PR. Thanks.